### PR TITLE
[LOOP-2546] Display onboarding at startup

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-2546
- https://tidepool.atlassian.net/browse/LOOP-3143
- https://tidepool.atlassian.net/browse/LOOP-3144
- https://tidepool.atlassian.net/browse/LOOP-3145
- Implement OnboardingUI protocol updates
- Fix Suspend Threshold Info view sometimes displaying twice

### CODE REVIEW

**Comment by darinkrauss** [comment]
At least two reviewers, please.

**Comment by darinkrauss** [comment]
LW PR for build: https://github.com/tidepool-org/LoopWorkspace/pull/440

File: LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift [hunk 1]
```diff
@@ -272,7 +266,14 @@ class OnboardingUICoordinator: UINavigationController, OnboardingViewController
     }
 
     private func stepFinished() {
-        if let nextScreen = currentScreen.next() {
+        var nextScreen: OnboardingScreen?
+
+        nextScreen = currentScreen.next()
+        if nextScreen == .nightscoutChooser && service?.isOnboarded == true {
```
Comment (rickpasetto) [inline LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift:275]: Hm...this special case is a bit odd and not obvious... Consider adding a code comment here as to why.

**Review by rickpasetto — APPROVED** [review]
One comment/suggestion but other than that LGTM 🦙

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**